src/webgpu/api/operation/adapter/requestDevice.spec.ts

```diff
@@ -157,6 +157,12 @@ g.test('stale')
           );
         }
         break;
+      case 'ConsumedAdapter':
+        // Cause an operation error by requesting a device on a consumed adapter.
+        if (!awaitInitialError) {
```

Comment:
`awaitInitialError` was just intended to control whether we `await` or not on the first requestDevice call before continuing the test. Why is it different in this case?

Reply:
My understanding is that it doesn't make sense to wait for requestDevice to fail as the adapter is not consumed at that point.

Reply:
The test is set up so that if `initialError !== undefined`, there's always two requestDevice calls, one in this switch, and one later. Now this parameter is controlling whether there's one or two.

To make this make more sense, the parameters should be renamed something like:

```
    u
      .combine('request1', ['None', 'Valid', 'TypeError', 'OperationError'])
      .expand('awaitRequest1', p => p.firstRequest === 'None' ? [false] : [true, false])
      .combine('awaitRequest2', [true, false])
```

Reply:
I've renamed parameters and adjusted code. PTAL